liquibase-couchbase/src/main/java/liquibase/ext/couchbase/statement/CreateQueryIndexStatement.java

```diff
@@ -34,11 +33,11 @@ public class CreateQueryIndexStatement extends CouchbaseStatement {
 
     @Override
     public void execute(ClusterOperator clusterOperator) {
-        Collection collection = clusterOperator.getBucketOperator(keyspace.getBucket())
-                .getCollection(keyspace.getCollection(), keyspace.getScope());
         CreateQueryIndexOptions options = CreateQueryIndexOptions.createQueryIndexOptions()
                 .deferred(deferred)
                 .numReplicas(numReplicas);
-        clusterOperator.getCollectionOperator(collection).createQueryIndex(indexName, fields, options);
+        clusterOperator.getBucketOperator(keyspace.getBucket())
+                .getCollectionOperator(keyspace.getCollection(), keyspace.getScope())
```

Comment:
Could we add a version of this method that consumes keyspace to get collection and scope from it?

Reply:
Yes, we can, but in bucket operator we don't need name of bucket, and I think getting explicitly via collection name and scope name is more readable. What you think?